pkg/utils/structured.go

```diff
@@ -45,3 +47,13 @@ func UnstructuredToStructured(unstructuredObj *unstructured.Unstructured, target
 
 	return nil
 }
+
+func ValidClient(c client.Client, cli dynamic.Interface) error {
```

Comment:
ValidClient => ValidateClient is  better

Reply:
updated